package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "ui-components",
+  "version": "0.0.1",
+  "description": "UI components for ManageIQ project",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack --watch",
+    "postinstall": "bower install && typings install && webpack && webpack --config webpack.vendor.config.js",
+    "build-docs": "jsdoc -c jsdoc-conf.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ManageIQ/ui-components.git"
+  },
+  "authors": [],
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">= 5.1.0",
+    "npm": ">= 3.8.1"
+  },
+  "devDependencies": {
+    "angular": "1.5.7",
+    "angular-animate": "^1.5.5",
+    "angular-bootstrap": "^0.12.2",
+    "angular-jsdoc": "^1.4.2",
+    "angular-ui-select": "^0.12.100",
+    "bootstrap-combobox": "^1.0.2",
+    "bower": "^1.7.9",
+    "bower-webpack-plugin": "^0.1.9",
+    "browser-sync": "2.13.0",
+    "browser-sync-spa": "1.0.3",
+    "browser-sync-webpack-plugin": "1.0.3",
+    "copy-webpack-plugin": "3.0.1",
+    "css-loader": "^0.23.1",
+    "event-stream": "^3.3.3",
+    "extract-text-webpack-plugin": "1.0.1",
+    "file-loader": "^0.9.0",
+    "html-webpack-plugin": "^2.15.0",
+    "imports-loader": "^0.6.5",
+    "jquery": "3.0.0",
```

Comment:
little thing, but jquery-3.1 maybe? Because of that whole 3.0 eats your exceptions thing..